libraries/aws_s3_buckets.rb

```diff
@@ -24,15 +24,37 @@ def initialize(opts = {})
     @table = fetch_data
   end
 
+  # FilterTable setup
+  filter_table_config = FilterTable.create
+  filter_table_config.add(:bucket_names, field: :bucket_name)
+  filter_table_config.add(:tags, field: :tags)
+  filter_table_config.connect(self, :fetch_data)
```

Comment:
Any plurals you touch with this update can you switch them over to the new FilterTable syntax?